src/main/java/Main.java

```diff
@@ -1,5 +1,113 @@
+                        JSONObject item = items.getJSONObject(j);
+                        String itemName = item.getString("name");
+                        String usage = item.getString("usage");
+                        thisItemMap.put(itemName, new Item(itemName, name, usage));
+                        itemMap.put(itemName, new Item(itemName, name, usage));
+                    }
+
+                    JSONArray nextRoomsJson = room.getJSONArray("connects to");
+                    String[] nextRooms = new String[nextRoomsJson.length()];
+
+                    for (int k = 0; k < nextRoomsJson.length(); k++){
+                        String nextName = nextRoomsJson.getString(k);
+                        nextRooms[k] = nextName;
+                    }
+                    roomMap.put(name, new Room(name, thisItemMap, nextRooms, script));
+                }
+
+                System.out.println(ANSI_BLUE + "\nWhat is your name?\n" + ANSI_RESET);
+
+                try {
+                    playerName = in.readLine();
+                    System.out.println(ANSI_BLUE + "\n Welcome, " + playerName +". \n Let's begin. Your goal is to get to "
+                            + endRoom.getString("name") + ".\n You are currently located at " + startRoom.getString("name")
+                            + " and your inventory is currently empty.\n Type " + ANSI_MAGENTA + "'help'" +ANSI_BLUE + " if you ever need help.\n" + ANSI_RESET);
+                }catch(Exception e){
+                    e.printStackTrace();
+                }
+
+                Player player1 = new Player(playerName,  new ArrayList<Item>(),roomMap.get(startRoom.getString("name")));
+                System.out.println(ANSI_BLUE + "\n" + startRoom.getString("script")  + "\n" + ANSI_RESET);
+
+                mainWhile: while(!player1.getCurrentRoom().getRoomName().equals(endRoom.getString("name"))){
+                    try {
+                        String[] input2 = in.readLine().split("\\s+");
+                        if(input2[0].equals("quit") && input2.length == 1) {
+                            System.out.println(ANSI_BLUE + "\nYou lose!\n" + ANSI_RESET);
+                            break mainWhile;
+                        }
+                        else {
+                            Action.doAction(input2, player1);
+                        }
+                    } catch(Exception e){
+                        e.printStackTrace();
+                    }
+                    if(player1.getCurrentRoom().getRoomName().equals(endRoom.getString("name"))){
+                        System.out.println(ANSI_BLUE + "\nCONGRATULATIONS, " + playerName +  "! You made it to " + endRoom.getString("name") + "\n" + ANSI_RESET);
+                    }
+                }
+            }else{
+                System.out.println(ANSI_BLUE + "\nNot a valid json file.\n" + ANSI_RESET);
+            }
+        } catch(Exception e) {
+            e.printStackTrace();
+        }
     }
 }
```

Comment:
-Instead of having three different functionalities i.e reading JSON content, generating the player and reading input all inside main, you could've split it up into other methods.
-Ok error handling

Overall, looks good to me.